src/main/java/net/sf/jabref/importer/fileformat/RepecNepImporter.java

```diff
@@ -186,12 +186,7 @@ public String getId() {
 
     @Override
     public String getDescription() {
-        return
-        "Imports a New Economics Papers-Message (see http://nep.repec.org)\n"
-                + "from the REPEC-NEP Service (see http://www.repec.org).\n"
-                + "To import papers either save a NEP message as a text file and then import or\n"
-                + "copy&paste the papers you want to import and make sure, one of the first lines\n"
-                + "contains the line \"nep.repec.org\".";
+        return "Imports a New Economics Papers-Message from the REPEC-NEP Service.";
```

Comment:
OK for @JabRef/developers, but in the future, a small reasoning should be kept or the @JabRef/stupro-qa  comment should be deleted.